src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const lowerCaseCompare = query.toLowerCase();
```

Comment:
Optionally: you can make it a function and compare `query` with `title` and `description` here

And use it below to avoid code duplication

```suggestion
   const lowerCaseCompare = (string: string) => (
    string.toLowerCase().includes(query.toLowerCase())
  );
```